sidecar/proxy/envoy/manager.go

```diff
@@ -565,60 +548,6 @@ func buildFS(ruleList []api.Rule, workingDir string) error {
 		return err
 	}
 
-	dirName, err := ioutil.TempDir(workingDir+runtimeVersionsPath, "")
-	if err != nil {
```

Comment:
If setting those values in the runtime path is configurable it is not documented [here](https://lyft.github.io/envoy/docs/configuration/http_filters/fault_filter.html).  we are not currently doing this for faults